Rakefile

```diff
@@ -23,16 +24,21 @@ task :tag_release do
 end
 
 desc "Run the unit test suite"
-task :default => 'test:units'
+task :default => 'test:local'
 task :test => 'test:units'
 
+RuboCop::RakeTask.new
```

Comment:
Looks good to me.  FWIW we went with https://github.com/activeadmin/activeadmin/pull/5174/files